### DISCUSSION TITLE
A better/easier way to share beatmaps

### DISCUSSION THREAD

**Discussion by RadioactivePeaches** [ppy/osu#27620] [Ideas] [answered]
Currently (best way I know how right now) on lazer you have to right click the beatmap you want to share, click details, and then click on the very small "View in browser" button, which opens the browser which you can then copy the link of to share it on discord, for example. 
I'd like to suggest a "Copy beatmap link" in a more accessible and visible area, like in the details section on the left (section with the scoreboards) so that the button is easier to find as well as easier to share.
Please do let me know if there is an existing easier way to do this, I hope my suggestion ends up useful somehow.

**Comment by Rekunan** [comment, marked as answer]
Duplicate of #26275
